modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -201,7 +201,7 @@ definition {
 
 	@description = "LPS-161917 - Verify that it's possible to create an e-mail notification template"
 	@priority = 5
-	test CanCreateEmailTemplate {
+	test CanCreateEmailNotificationTemplate {
```

Comment:
Why is this change part of 6092c70f81e716166f882b70e997533edd337b52?

Reply:
I put on [66c8157833893e45385de72ed5386d0f90197590](https://github.com/liferay-bpm-qa/liferay-portal/pull/828/commits/66c8157833893e45385de72ed5386d0f90197590) since it's in the same context.